doc/team_practices.md

```diff
@@ -0,0 +1,37 @@
+# C2 Team Practices
+
+This document outlines best practices for the C2 team, and anyone wanting to contribute to the project.
+
+## Workflow
+
+All code changes should originate in a story on the [story tracker](https://pivotaltracker.com/n/projects/1149728).
```

Comment:
I would delineate those as 2 different cases:(1) the refactor that occurs because "hey, I could improve that while I'm doing this" (in which case there is already a Story behind "this") and (2) the refactor that is a Chore that needs time set aside. The recent Roles refactor comes under the latter category, imo.

So I wouldn't expect the first case to need a Story: it's being tracked already, indirectly, in the time it takes to implement the originating Story. The second case needs a Story, so we can schedule for it.